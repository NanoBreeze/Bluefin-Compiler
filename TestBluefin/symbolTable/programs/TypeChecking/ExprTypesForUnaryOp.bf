int a;
bool b;

int main() {
	-a;
	if (!b) {}
}