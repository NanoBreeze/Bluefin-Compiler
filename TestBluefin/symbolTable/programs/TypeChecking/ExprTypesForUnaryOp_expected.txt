a - t_int
-a - t_int
b - t_bool
!b - t_bool
